app/services/application_state_change.rb

```diff
@@ -29,6 +29,7 @@ def initialize(application_choice)
     state :unsubmitted do
       event :submit, transitions_to: :awaiting_references
       event :send_to_provider, transitions_to: :awaiting_provider_decision
+      event :cancel, transitions_to: :cancelled
```

Comment:
I wonder if this should be either it's own state (`never_submitted`) or `application_not_sent` (https://github.com/DFE-Digital/apply-for-teacher-training/pull/2840). What do you think?

Reply:
I think `application_not_sent` would make more sense. We don't want to get these mixed up with 'real' cancelled applications from a reporting point of view. I don't _think_ we need a brand new state because these applications would be easily distinguished from the other `application_not_sent` ones as they are un-submitted.

The only reason for going with `cancelled` was to be consistent with https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/services/send_applications_to_provider.rb#L11 . Maybe we should change that too?